lib/rails_admin/adapters/active_record/association.rb

```diff
@@ -39,6 +39,7 @@ def foreign_key
 
         def foreign_key_nullable?
           return if foreign_key.nil? || type != :has_many
+          return true if association.through_reflection
           klass.columns_hash[foreign_key.to_s].null
```

Comment:
Are you sure that above fix is safe also for polymorphic association?
Isn't it safe to just nil-guard `null` call?
```ruby
klass.columns_hash[foreign_key.to_s].try(:null)
```